docker-compose-localstack.yaml

```diff
@@ -9,10 +9,8 @@ services:
       - "4566-4597:4566-4597"
     environment:
       - SERVICES=sqs,sns
-      - DATA_DIR=/tmp/localstack/data
```

Comment:
localstack wouldn't start for me so I followed instructions in https://docs.localstack.cloud/getting-started/installation/#docker-compose